src/Commands/Env/CodeLogCommand.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Env;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+class CodeLogCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+
+    /**
+     * Show an environment's code log.
+     *
+     * @command env:code-log
+     *
+     * @param string $site_env Site & environment to show log for.
```

Comment:
I've just noticed that there aren't any labels on the fields.  Here's the syntax for quick reference:
https://github.com/pantheon-systems/terminus/blob/master/src/Commands/Upstream/UpdatesListCommand.php#L20-L26